Product/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+
+class MainCategory(models.Model):
+	name  = models.CharField(max_length=45)
+
+	class Meta:
+	    db_table = 'main_categories'
+
+class Category(models.Model):
+	name          = models.CharField(max_length = 45)
+	main_category = models.ForeignKey(MainCategory, on_delete=models.CASCADE)
+
+	class Meta:
+	    db_table = 'categories'
+
+class SubCategory(models.Model):
+	name          = models.CharField(max_length = 45)
+	category      = models.ForeignKey(Category, on_delete =models.CASCADE)
+
+	class Meta:
+	    db_table = 'sub_categories'
+
+
+class ProductGroup(models.Model):
+	name  = models.CharField(max_length = 45, null=True)
+
+	class Meta:
+	    db_table = 'product_group'
+
+class Products(models.Model):	
```

Comment:
```suggestion
class Product(models.Model):	
```
클래스 이름 단수로 지정해주세요.
Products 변수가 사용된 다른 곳들도 수정하셔야 합니다 :)